### ISSUE TITLE
Update bulk test cases for JDK12

### ISSUE THREAD

**Issue by matozoid** [javaparser/javaparser#2007]

**Comment by MysterAitch** [comment]
I'm a little confused about what this means -- could you expand on this a little please @matozoid ?

**Comment by matozoid** [comment]
Yeah, didn't mention this one yet. Take a look at BulkParseTest.java. It is documented :-)
The point of it is to figure out how compliant the parser is right now. The bulk test case is a copy of some OpenJDK source code which they use to test for bugs, features, and general weirdness. When run it will update a file with all the results, and the changes have to be inspected. Do we expect a failure here (because the flie being read is a test case that should give a compile error) or is it something we missed in the grammar?

By now it should be updated to JDK14 or 15 or whatever we're using nowadays of course :-)

**Comment by MysterAitch** [comment]
I've not been able to find any langtools for any recent versions of java... 

Following instructions I can see that the website goes up to JDK 10, but beyond that they're all in a JDK-Updates folder with no langtools available.

![image](https://user-images.githubusercontent.com/1537214/100552533-ac6d1f00-327f-11eb-8670-49abea610f23.png)

JDK9 - 13.2 MiB
http://hg.openjdk.java.net/jdk9/jdk9/langtools/archive/65bfdabaab9c.zip

JDK10 - 11.8 MiB
http://hg.openjdk.java.net/jdk10/jdk10/langtools/archive/19293ea3999f.zip


Here's the listing at http://hg.openjdk.java.net/ :
![image](https://user-images.githubusercontent.com/1537214/100552547-d0c8fb80-327f-11eb-9963-d2c17aba9d3b.png)


Downloading any of the zips from java-updates seems to download much bigger source files 

e.g. JDK15 - 163 MiB
http://hg.openjdk.java.net/jdk-updates/jdk15u/archive/ac639af55573.zip

**Comment by matozoid** [comment]
Is this it? https://github.com/openjdk/jdk/tree/master/test/langtools

**Comment by MysterAitch** [comment]
It appears to be!

and it seems different versions can be gotten via the git tags like so:

https://github.com/openjdk/jdk/tree/jdk-15+36/test/langtools

Not quite sure if that can be downloaded as a zip in isolation though.

**Comment by MysterAitch** [comment]
Ah, just realised from your git link that the langtools are now in the test directory, and that prompted me to look for the same paths on the mercurial repo. 

Turns out that it is visible online at e.g.: http://hg.openjdk.java.net/jdk-updates/jdk15u/file/ac639af55573/test/langtools

Again, not sure how that directory can be downloaded in isolation though.

**Comment by matozoid** [comment]
It probably can't? Then the bulk test is getting bulkier :-)
